main.cpp

```diff
@@ -3,6 +3,7 @@
 #include "render_context.hpp"
 #include "effect_player.hpp"
 #include "camera_utils.hpp"
+#include "libraries/utils/glfw_window.hpp"
```

Comment:
and i think this line is superfluous because the "render_context.hpp" also includes this file